parsec/backend/asgi/rpc.py

```diff
@@ -240,15 +273,71 @@ async def anonymous_api(raw_organization_id: str) -> Response:
         _handshake_abort_bad_content(api_version=api_version)
 
     # Run command
-    rep = await cmd_func(client_ctx, msg)
+    try:
+        rep = await cmd_func(client_ctx, msg)
+    except Exception as exc:
+        print("rpc didn't handle this exception:", type(exc))
+        raise exc
     return _rpc_msgpack_rep(rep, api_version)
 
 
+@rpc_bp.route("/invited/<raw_organization_id>/<raw_invitation_token>", methods=["POST"])
+async def invited_api(raw_organization_id: str, raw_invitation_token: str) -> Response:
+    backend: BackendApp = g.backend
+
+    api_version, client_api_version, organization_id, _, _, _, invitation = await _do_handshake(
+        raw_organization_id=raw_organization_id,
+        backend=backend,
+        allow_missing_organization=False,
+        check_authentication=False,
+        expected_content_type=CONTENT_TYPE_MSGPACK,
+        expected_accept_type=None,
+        raw_invitation_token=raw_invitation_token,
+    )
+
+    # Unpack verified body
+    body: bytes = await request.get_data(cache=False)
+    try:
+        msg = unpackb(body)
+    except SerdePackingError:
+        _handshake_abort_bad_content(api_version=api_version)
+
+    cmd = msg.get("cmd")
+    if not isinstance(cmd, str):
+        _handshake_abort_bad_content(api_version=api_version)
+
+    try:
+        cmd_func = backend.apis[ClientType.INVITED][cmd]
+    except KeyError:
+        _handshake_abort_bad_content(api_version=api_version)
+
+    assert invitation is not None
+    client_ctx = InvitedClientContext(
+        api_version=api_version,
+        client_api_version=client_api_version,
+        organization_id=organization_id,
+        invitation=invitation,
+    )
+    client_ctx.logger.info(
+        f"Invited client successfully connected (client/server API version: {client_api_version}/{api_version})"
+    )
+
+    try:
+        cmd_rep = await cmd_func(client_ctx, msg)
+    except CloseInviteConnection:
+        _handshake_abort(410, api_version=api_version)
```

Comment:
What is  this HTTP 410 error ? it must be documented in the comment line 83

Reply:
Done in #4107